ttkwidgets/font/propertiesframe.py

```diff
@@ -2,18 +2,47 @@
 Author: RedFantom
 License: GNU GPLv3
 Source: This repository
+
+Edited by rdbende: translation and default font properties option
 """
 # Based on an idea by Nelson Brochado (https://www.github.com/nbro/tkinter-kit)
 import tkinter as tk
 from tkinter import ttk
+from locale import getdefaultlocale
+
+# --- Translation
+EN = {"Font properties:": "Font properties:"}
+
+FR = {"Font properties:": "Propriétés"}
+
+DE = {"Font properties:": "Eigenschaften"}
+
+HU = {"Font properties:": "Betűstílus"}
```

Comment:
It would be good imo to create a utility class to provide translations and internationalization, so that it can be used across the entire lib, as well as be editable by the programmer.

I'll make a separate PR about this to add such a functionnality.